config.default.yaml

```diff
@@ -312,18 +312,20 @@ costs:
 
 
 monte_carlo:
+  add_to_snakefile: true
   options:
-    add_to_snakefile: false
-    samples: 7    # number of optimizations
-    sampling_strategy: "chaospy"    # "pydoe2", "chaospy", "scipy", packages that are supported
+    method: "global_sensitivity"  # or any_chance_store_test
```

Comment:
The alternative method may have a name not focused on store. "single_best_in_worst"?
I like the single_best_in_worst_list or something like that.

Reply:
Description of what the methods are is necessary maybe in the documentation and also a little comment here

Reply:
Done